Index: framework/webapp/config/fop.xconf
===================================================================
--- framework/webapp/config/fop.xconf   (revision 1785051)
+++ framework/webapp/config/fop.xconf   (working copy)
@@ -31,7 +31,7 @@
 <fop version="1.0">

   <!-- Base URL for resolving relative URLs -->
-  <base>http://localhost:8080</base>
+  <base>https://demo-trunk.ofbiz.apache.org</base>

   <!-- Source resolution in dpi (dots/pixels per inch) for determining the size of pixels in SVG and bitmap images, default: 72dpi -->
   <source-resolution>72</source-resolution>
